### PR TITLE
38 update issue and PR templates

### LINKED ISSUE #38: Create a GitHub PR and issue template

### PR DESCRIPTION
**Pull request by wyli**
Fixes #38

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
```
Comment (Nic-Ma) [inline .github/ISSUE_TEMPLATE/bug_report.md]: What's the difference between "name" and "title"?
Suggest to change "name" to "category".

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+2. Install '....'
+3. Run commands '....'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (Nic-Ma) [inline .github/ISSUE_TEMPLATE/bug_report.md]: 1. I think maybe "Environment" is more suitable thank "Desktop".
2. Also need to include the hardware information, especially GPU cards type.
3. And add below software information:
    (1) Python version.
    (2) PyTorch version.
    (3) Ignite version.
    (4) CUDA version(optional).
    (5) cuDNN version(optional).

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
```
Comment (Nic-Ma) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Same as bug report, suggest to "type".

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (wyli) [reply .github/ISSUE_TEMPLATE/bug_report.md]: That table with keywords is actually autogenerated via github GUI, I wouldn't change that for now.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (wyli) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks, I've updated the section. hope it's fine to keep it light-weight for now. in the future we should use a CLI script to automatically collect/show system info.

**Pull request merged**